### ISSUE TITLE
allow subject to be referenced in multiple maps in a transaction

### ISSUE THREAD

**Issue by dpetran** [fluree/db#241]

**Comment by mpoffald** [comment]
@dpetran to provide example/specification

**Comment by dpetran** [comment]
@aaj3f Do you remember the use case this describes? I'm trying to recreate, but it's not giving me any trouble. This is my tx json:

```
{"@context"  {"ex"  "http://example.com/"}
                            "@graph"
                            [{"@id"  "ex/person",
                              "ex/name"  "Geoff",
                              "ex/book"  [{"@id"  "ex/book1"}, {"@id"  "ex/book2"}]},
                             {"@id"  "ex/book1",
                              "ex/name"  "Foo"},
                             {"@id"  "ex/book2",
                              "ex/name"  "Bar"}]}
```

**Comment by bplatz** [comment]
This is duplicated - and already fixed in PR #302
